packages/flutter/lib/src/rendering/object.dart

```diff
@@ -741,16 +740,17 @@ typedef LayoutCallback<T extends Constraints> = void Function(T constraints);
 /// [PipelineOwner] for the render tree from which you wish to read semantics.
 /// You can obtain the [PipelineOwner] using the [RenderObject.owner] property.
 class SemanticsHandle {
-  SemanticsHandle._(this._owner, this.listener)
-      : assert(_owner != null) {
+  SemanticsHandle._(PipelineOwner owner, this.listener)
+      : assert(owner != null),
+        _owner = owner {
     if (listener != null)
-      _owner.semanticsOwner.addListener(listener);
+      _owner!.semanticsOwner!.addListener(listener!);
   }
 
-  PipelineOwner _owner;
+  PipelineOwner? _owner;
```

Comment:
why nullable?

Reply:
It is set to null in dispose

Reply:
if that's the only reason, we could remove that line and make it non-nullable